src/Util/Exchange.php

```diff
@@ -142,7 +142,7 @@ public function getPayLoad()
             } else {
                 $rawBody = file_get_contents('php://input');
                 if (empty(trim($rawBody))) {
-                    throw new Exception('Empty payload', 8002);
+                    throw new Exception('Empty or incomplete payload', 8002);
                 }
 
                 $tguData = json_decode($rawBody, true, 512, 4194304);
```

Comment:
Why `4194304`? It doesn’t appear to match any documented values. If it’s a combination of multiple flags, consider using the named constants instead, e.g., `flags: JSON_BIGINT_AS_STRING | JSON_OBJECT_AS_ARRAY`, for better readability.  

Also, if `4194304` can be removed, note that `512` is the default value and could be omitted as well.